data_steward/utils/deactivated_participants.py

```diff
@@ -0,0 +1,98 @@
+"""
```

Comment:
I suggest renaming this file.  The purpose of building the utility is to make something reusable.  This file name is extremely specific.  We will probably need to visit the ParticipantSummary API more for other status types in the future, i.e. `deceased`, `withdrawn`, etc.  

I suggest naming it something like `participant_summary_requests.py`.  This would allow a natural place to add that type of information.

I've made some specific comments in here but have a larger structural suggestion.  I would refactor some of the functionality out of `get_deactivated_participants`.  Specifically, the loop where you are getting your data.  You could move that loop into a helper function that could be use by `get_deactivated_participants` now and by some other participant summary api reader function in the future.  If you would like to, we can have a call and discuss it.

Reply:
I understand moving the loop fetching the data out and into it's own function. I wasn't aware that this utility was going to do more than just fetch deactivated participants. It makes sense to rename the file along with moving the fetching data loop as it could be used in various functions that would get other types of data like you mentioned.